### PR TITLE
YO-1447: first implementation

### PR DESCRIPTION
**Pull request by lwithers**
@lampkicking/serverside

### CODE REVIEW

File: client/client.go [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+	"image"
+
+	"github.com/lampkicking/ocr-go/ocr"
+)
+
+// Client objects provide concurrency-safe methods to request the
+// passport-ocr-server perform optical character recognition on images of
+// passports.
+//
+// Remember that passportocr.Characters implements fmt.Stringer, allowing
```
Comment (LKDraytonBrown) [inline client/client.go]: typo? "passport ocr.Characters"

File: itest/tests/init.go [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+	if err != nil {
+		return err
+	}
+	dec, _, err = image.Decode(bytes.NewReader(EncImgMrzYuv))
+	if err != nil {
+		return fmt.Errorf("mrz-johnmills.yuv: %v", err)
+	}
+	DecImgYuv = dec.(*image.YCbCr)
+
+	EncImgMrzJpeg, err = ioutil.ReadFile(path.Join(TestDataDir,
```
Comment (LKDraytonBrown) [inline itest/tests/init.go]: This 'ioutil.ReadFile(path.Join(TestDataDir,<file name>))' is repeated, maybe petty, but thought I'd comment.

**Comment by LKDraytonBrown** [comment]
Other than the two very minor comments, this looks good to me.

**Comment by lwithers** [comment]
Updated to cope with platform-layer-go/client→call rename.

Depends on:
* https://github.com/lampkicking/docker-scripts/pull/2
* https://github.com/lampkicking/yoti-backend/pull/21
* https://github.com/lampkicking/ansible-deployment/pull/42

**Pull request merged**